### PR TITLE
Favicon fix and acrd logo inclusion

### LINKED ISSUE #60: Missing favicon
The favicon is missing and needs to be added

### PR DESCRIPTION
**Pull request by suneel87**
Signed-off-by: Suneel Nadipalli <nsuneel89@gmail.com>

Fixes #60 
- Added Aykya logo favicon to all pages
- Added "In association with" arcd logo to home page and footer

### CODE REVIEW

**Comment by suneel87** [comment]
@aravindvnair99 @VinayakTekade I've added the favicon and included the ACRD logo wherever required

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@suneel87 Fix the issues pointed out by @AccessLint

File: public/meta.ejs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<meta charset="UTF-8" />
```
Comment (aravindvnair99) [inline public/meta.ejs:1]: EJS files can't be in public

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: public/meta.ejs (hunk 1 above)
Comment (suneel87) [reply public/meta.ejs]: > EJS files can't be in public

@aravindvnair99 Done, fixed accesslint's issuses too

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@suneel87 This doesn't look right:

image

**Comment by suneel87** [comment]
> @suneel87 This doesn't look right:
> 
> image

@aravindvnair99 http://0fcf5108b819.ngrok.io/ Check it once here, please

**Comment by suneel87** [comment]
@aravindvnair99 , I think I fixed the issues, could you check the link again, http://0fcf5108b819.ngrok.io/ ?

**Comment by aravindvnair99** [comment]
> @aravindvnair99 , I think I fixed the issues, could you check the link again, http://0fcf5108b819.ngrok.io/ ?
> 
> > **[AYKYA](http://0fcf5108b819.ngrok.io/)**

@suneel87 It's the same:

image

**Comment by suneel87** [comment]
> > @aravindvnair99 , I think I fixed the issues, could you check the link again, http://0fcf5108b819.ngrok.io/ ?
> > > **[AYKYA](http://0fcf5108b819.ngrok.io/)**
> 
> @suneel87 It's the same:
> 
> image
> 
> > **[AYKYA](http://0fcf5108b819.ngrok.io/)**

@aravindvnair99 , it's working fine for me and Lakshmi too, I'm not sure what's happening at your end. Should I check with everyone else in the group too?

**Comment by suneel87** [comment]
@aravindvnair99 All right, I've added a top margin to the countdown, I think that should apply everywhere(for all systems),could you check again?
http://0fcf5108b819.ngrok.io/

**Comment by aravindvnair99** [comment]
> > > @aravindvnair99 , I think I fixed the issues, could you check the link again, http://0fcf5108b819.ngrok.io/ ?
> > > > **[AYKYA](http://0fcf5108b819.ngrok.io/)**
> > 
> > 
> > @suneel87 It's the same:
> > image
> > > **[AYKYA](http://0fcf5108b819.ngrok.io/)**
> 
> @aravindvnair99 , it's working fine for me and Lakshmi too, I'm not sure what's happening at your end. Should I check with everyone else in the group too?
> 
> > **[AYKYA](http://0fcf5108b819.ngrok.io/)**

@suneel87 This is a common thing while developing, what works on one developer's machine doesn't necessarily have to work on another. We have different screen resolutions and density.

**Review by aravindvnair99 — APPROVED** [review]

**Comment by aravindvnair99** [comment]
> @aravindvnair99 All right, I've added a top margin to the countdown, I think that should apply everywhere(for all systems),could you check again?
> http://0fcf5108b819.ngrok.io/
> 
> > **[AYKYA](http://0fcf5108b819.ngrok.io/)**

@suneel87 Yes, it's proper now.

**Pull request merged**